Readme.md

```diff
@@ -1,43 +1,440 @@
-# Build the image
-We want to provide a keycloak image with oid4vci feature enabled.
+# Keycloak as a Verifiable Credential Issuer with OID4VCI
 
+This guide walks you through configuring Keycloak to issue Verifiable Credentials (VCs) using the OpenID for Verifiable Credential Issuance (OID4VCI) protocol.
 
-For the moment, a script sequence leads the developer through steps needed to deploy keycloak and have it issue a verifiable credential
+# TLDR
 
-For this version: checkout the branch: https://github.com/adorsys/keycloak-oid4vc/tree/issue-30525. 
+Checkout this project.
 
-This is subject of the pull request: https://github.com/keycloak/keycloak/pull/30692. You will go back to KC upstream as soon as this branch is merged.
+## Checkout, Build and Deploy keycloak
 
-# Steps
-- Checkout this project.
-- In the project directory execute following scripts:
+### Prerequisites
+
+Before proceeding, ensure you have the following tools installed on your system:
+
+* **OpenSSL:** A command-line tool for working with SSL/TLS certificates, keys, and other cryptographic functions.
+* **Keytool:** A Java key and certificate management utility included with the Java Development Kit (JDK).
+
+**Verification:** You can verify that the tools are working by running:
+
+```bash
+openssl version
+keytool -version
```

Comment:
Tried this command, but it doesn't work
I propose we change it to
```bash
keytool -h
```

Reply:
you mean openssl -h

Reply:
both work on my machine